_posts/2021-05-24-ols-3-participant-katharina-kloppenborg-speedblog.md

```diff
@@ -0,0 +1,55 @@
+
+# What else did I learn along the way?
+- **Learning by doing:** It is easier to internalize how to apply open science practices by actually implementing it in a project, which is why I think that OLS is taking a great approach!
+- **‘Open’ can equal ‘more accessible’:** Preparing my project to be open for contributions made me reflect the understandability, and accessibility of my project, as well as the value that could motivate others to visit the blog and contribute. I think this process helped me to improve my project in general.
+- **Simple can be hard:** I also realized how hard it is to communicate the goals and processes of my project in an understandable way, which in turn helped me to understand my project better myself. However, I realized that there is still a lot that can be improved.
+- **The why and the how:** Starting the project by breaking it down to a vision statement, a roadmap, user and contributor personas helped me to get a clearer picture of what I wanted to achieve, and how.
+- **Finding contributors is not self-evident:** Creating a project according to open science principles is one necessary step towards participation, but it does not mean that there will be many contributions from other people from the start. Many projects start with one or a few persons, and other contributors might join along the way when the project is already running. Open science is also about reusability and transparency, and a good way to practice science in general, so one should not get discouraged if there are no or only a few contributions of other people in the beginning. 
+- **We’re not alone out there✨:** Finally, the OLS program showed me how many people are interested in practicing open science and that there is a great community with lots of practical knowledge for these kinds of endeavors.
+
+# Next steps and long term tasks
+As a next step I will try to get more feedback on my website as it is now, knowing that there is still a lot to do (feel free to leave some comments 💖). I will also try to develop a habit of adding blog articles when I stumble upon interesting peer production features. The tagging system and template might have to be adapted little by little according to changing focus and needs along the way. An important step will also be to spread the word to invite possible users and contributors: For now, I plan to do this via a formal publication (of which the blog project will be a supplementary material), as well as a conference poster of the project. With the project growing, I hope to be able to build a useful resource and an accessible and modular way to collect and communicate my own research findings, and would be glad if other people would join me on the way!
```

Comment:
Where do you want people to leave comments? On a GitHub repository? Do you have an URL?

Reply:
Good point! Yes, comments on the project's github repository would be good (https://github.com/PeerProducedResearch/design-practices). The repository is also linked on the project website, but it is true that it is better to be more explicit. I will also add the link the the preprint of my publication in this section, just noticed that I hadn't updated the text with that yet :)

Reply:
I added the info in a new commit